content/guides/printing/overview.adoc

```diff
@@ -0,0 +1,59 @@
+= Overview
+Sameer Rahmani
+2016-09-07
+:type: guides
+:icons: font
+:status: published
+
+ifdef::env-github,env-browser[:outfilesuffix: .adoc]
+
+In this series of articles we're going to take a look at different ways of printing in Clojure.
```

Comment:
It seems weird to start with pprint. Rather than commit to a series with a single article, I would just put it under guides and we will organize if/when it expands.